incrementExact docs added (#1140)
Codecademy/docs: content/java/concepts/math-methods/terms/incrementExact/incrementExact.md

---

> Title: '.incrementExact()'
> 
> Description: 'Returns the argument incremented by one, throwing an exception if the result overflows the datatype.'
> Subjects:
> - 'Computer Science'
>   Tags:
> - 'Functions'
> - 'Methods'
> - 'Arithmetic'
> 
>   CatalogContent:
> - 'learn-java'

Reviewer: ```suggestion
Title: '.incrementExact()'
Description: 'Returns the argument incremented by one, throwing an exception if the result overflows the data type.'
Subjects:
- 'Computer Science'
  Tags:
- 'Functions'
- 'Methods'
- 'Arithmetic'
  CatalogContent:
- 'learn-java'
```

Author: Removing these extra lines will make the content overlap. Added screenshot. Please comment
<img width="1228" alt="Screenshot 2022-10-21 at 4 22 55 PM" src="https://user-images.githubusercontent.com/6908935/197327884-049a7bbc-9f5e-40f4-ae70-566fd9ae589d.png">

Reviewer: Not sure where you are previewing the Markdown, but when I preview on GitHub, the metadata renders expected w/ the suggested changes:

<img width="1382" alt="Screen Shot 2022-10-24 at 7 41 41 AM" src="https://user-images.githubusercontent.com/23144845/197517896-363d9c89-6af9-4455-b8b9-fdab807aa4ca.png">

Reviewer: You can certainly go off an published entry's meta-data to get a sense of what we're looking for: https://github.com/Codecademy/docs/edit/main/content/java/concepts/math-methods/terms/decrementExact/decrementExact.md

Author: @Dusch4593 , Thank you for guiding me and sharing the screenshot. I have made the necessary changes. can you please take a look.

---

> The **`Math.incrementExact()`** method returns the argument incremented by one or throws exception if the result overflows the specified datatype either long or int.

Author: Changes done.

> The **`Math.incrementExact()`** method returns the argument incremented by one. It throws an [exception](https://www.codecademy.com/resources/docs/java/errors) if the result overflows the specified datatype either `long` or `int`.

Reviewer: ```suggestion
The **`Math.incrementExact()`** method returns the argument incremented by one. It throws an [exception](https://www.codecademy.com/resources/docs/java/errors) if the result overflows the specified [data type](https://www.codecademy.com/resources/docs/java/data-types) (`long` or `int`).
```

## Syntax

```pseudo
Math.incrementExact(num)
```

> - The `num` argument is a `long` or `int` value.

Reviewer: ```suggestion
The `num` parameter is of type `long` or `int`.
```

## Example

> The following example uses `Math.incrementExact()` to increases the input by 1:

Author: Changes done.

> The following example uses `Math.incrementExact()` to increase the input by 1:

Reviewer: ```suggestion
The following example uses the `.incrementExact()` method to increase the input by 1:
```

```java

> public class Main {

Reviewer: ```suggestion
// Main.java

public class Main {
```
